apps/Vision/HeatMatrix.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from queue import Queue
+
+class HeatMatrix:
+
+   def __init__(self, size, queue_size, origin=None):
+      if size <= 1:
```

Comment:
Is size supposed to be a single int, leading to a square matrix? Or should this be modelled to the dimensions more similar to an image frame, for example (9, 16)?